shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -13,7 +13,9 @@ static constexpr char kChannelName[] = "flutter/platform";
 
 PlatformChannel::PlatformChannel(flutter::BinaryMessenger* messenger)
     : channel_(std::make_unique<flutter::MethodChannel<rapidjson::Document>>(
-          messenger, kChannelName, &flutter::JsonMethodCodec::GetInstance())) {
+          messenger,
+          kChannelName,
+          &flutter::JsonMethodCodec::GetInstance())) {
```

Comment:
You may follow the clang-format's suggestion unless there's a specific reason.

Reply:
This modification was done automatically, when I applied VSCode settings you suggested in comment https://github.com/flutter-tizen/engine/pull/65#issuecomment-820894094
I've also tried to use formatting script
dart ./ci/bin/format.dart --fix --no-all-files
which also applies such formatting on this line.

Please let me know what policy should be applied:
* do not format already existing code, just own modification
* do apply suggestions from clang formatter even not in the code you change, as I did in this case

Reply:
I'm not sure why your formatter works differently from mine but you can just leave as it is.